sidechain/consensus/slots/Cargo.toml

```diff
@@ -23,6 +23,7 @@ sp-consensus-slots = { version = "0.10.0-dev", default-features = false, git = "
 sp-runtime = { version = "6.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.19"}
 
 # local deps
+itp-settings = { path = "../../../core-primitives/settings", default-features = false }
```

Comment:
Can you add this to the std feature below `"itp-settings/std"`?

Reply:
settings does not have an `std` feature, for once I checked when adding this dependency 😄

Reply:
In this case - may I ask you to remove  `default-features = false` completely? 😄

Reply:
I can do that 👍